server/protocols/jmap/src/main/java/org/apache/james/jmap/crypto/SignedTokenManager.java

```diff
@@ -24,25 +24,28 @@
 
 import javax.inject.Inject;
 
-import org.apache.james.jmap.api.ContinuationTokenManager;
+import org.apache.james.jmap.api.SimpleTokenManager;
+import org.apache.james.jmap.model.AttachmentAccessToken;
 import org.apache.james.jmap.model.ContinuationToken;
+import org.apache.james.jmap.model.SignedExpiringToken;
 import org.apache.james.util.date.ZonedDateTimeProvider;
 
 import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
 
-public class SignedContinuationTokenManager implements ContinuationTokenManager {
+public class SignedTokenManager implements SimpleTokenManager {
```

Comment:
I think you mixed factory and signing responsibilities.
What about having only signing methods here and write a factory that handle token creations ?